app.js

```diff
@@ -17,8 +19,15 @@ app.use(favicon(path.join(__dirname, "public", "favicon.ico")));
 app.use(logger("dev"));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
-app.use(cookieParser());
-app.use(session({ secret: "averyrandomkeyfromgotofail" }));
+app.use(session({
+    store: new sqliteSessionStore,
+    secret: "averyrandomkeyfromgotofail",
+    name: "gotofail.sid",
+    sameSite: true,
+    saveUnitialized: true,
+    rolling: true,
+    cookie: { maxAge: 30 * 24 * 60 * 60 * 1000 },
```

Comment:
I know this is a hundred thousand times more readable, but it did make me laugh a little.